### PR TITLE
Houssame work in progress

### PR DESCRIPTION
**Pull request by HoussameH**
Added orientation and tracking to the calc_CF_solar
Regression coefficients input folder moved to the last timestamp defined used.
PV Timeseries have an orientation parameter compatible with regression coefs.
Possibility to select which hub_heights or orientations to use for the regression (By default it uses all the available Timeseries for that technology).
Removed the Net files from generate_weather_files
Print technology name at the start of the script

### CODE REVIEW

File: Master.py [hunk 1]
```diff
@@ -822,9 +784,14 @@ def reporting(paths, param, tech):
 
     # Define sampling for sorted lists
     sampling = param["report_sampling"]
-
+    status = 0
     # Loop over each region
     for reg in range(0, nRegions):
+
+        # Display Progress
+        status += 1
```
Comment (kais-siala) [inline Master.py]: how does this work?
isn't status = reg +1?

File: Master.py [hunk 2]
```diff
@@ -966,7 +932,7 @@ def find_locations_quantiles(paths, param, tech):
         I_old = np.argsort(X)
 
         # Escape loop if intersection only yields NaN
-        if sum(np.isnan(X).astype(int)) == len(X):
+        if np.isnan(X).all():
```
Comment (kais-siala) [inline Master.py]: I like it :-)

File: Master.py [hunk 3]
```diff
@@ -1089,27 +1055,42 @@ def regression_coefficient(paths, param, tech):
...
         return
-    elif len(inputfiles) == 1:
-        hub_heights = np.zeros(1, dtype=int)
     else:
-        hub_heights = np.zeros(len(inputfiles), dtype=int)
+        settings = np.zeros(len(inputfiles), dtype=int)
 
         h = 0
         for filename in inputfiles:
-            heigh = int(filename.replace(paths[tech]["TS_height"] + '_', '').replace('_TS_' + str(param["year"])+ '.csv', ''))
```
Comment (kais-siala) [inline Master.py]: don't you mean height instead of heigh?

File: Master.py (hunk 1 above)
Comment (HoussameH) [reply Master.py]: Status is just a counter, I compute what is the total number and count to it.  You are right here I could use reg +1, but I'd like to keep using status as it is standard in other parts of the script as well as the generate_model script

File: Master.py (hunk 3 above)
Comment (HoussameH) [reply Master.py]: Yes, it was a typo. Now that we account for the orientation of the PV too, I call it settings or set. The new name might be a bit confusing, any ideas ?

**Pull request merged**